genai-perf/genai_perf/metrics/llm_metrics.py

```diff
@@ -26,22 +26,32 @@
 # (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
 # OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
-from typing import List
+from typing import List, Union
 
 from genai_perf.metrics.metrics import MetricMetadata, Metrics
 
 
 class LLMMetrics(Metrics):
     """A simple dataclass that holds core LLM performance metrics."""
 
-    LLM_REQUEST_METRICS = [
+    LLM_REQUEST_TIME_METRICS = [
         MetricMetadata("time_to_first_token", "ms"),
         MetricMetadata("inter_token_latency", "ms"),
+    ]
+
+    LLM_REQUEST_THROUGHPUT_METRICS = [
         MetricMetadata("output_token_throughput_per_request", "tokens/sec"),
-        MetricMetadata("output_sequence_length", "tokens"),
-        MetricMetadata("input_sequence_length", "tokens"),
     ]
 
+    LLM_REQUEST_METRICS = (
+        LLM_REQUEST_TIME_METRICS
+        + LLM_REQUEST_THROUGHPUT_METRICS
+        + [
+            MetricMetadata("output_sequence_length", "tokens"),
```

Comment:
You should make a separate variable to hold LLM_REQUEST_OTHER_METRICS (including the other metrics e.g. output sequence length), so that we keep the abstraction level the same. Then, this variable can combine all three of those variables.